examples/menubar/main.rs

```diff
@@ -0,0 +1,56 @@
+use gtk::prelude::*;
+use gtk::{gio, glib};
+
+fn main() {
+    // Create a new application
+    let application = gtk::Application::new(
+        Some("com.github.gtk-rs.examples.menubar"),
+        Default::default(),
+    );
+    application.connect_startup(configure_application);
+    application.connect_activate(build_ui);
+    application.run();
+}
+
+fn configure_application(app: &gtk::Application) {
+    let about = gio::SimpleAction::new("about", None);
+    about.connect_activate(|_, _| {
+        println!("About was pressed");
+    });
+    app.add_action(&about);
+
+    let quit = gio::SimpleAction::new("quit", None);
+    quit.connect_activate(glib::clone!(@weak app => move |_action, _parameter| {
+        app.quit();
+    }));
+    app.add_action(&quit);
```

Comment:
```suggestion
let about = gio::ActionEntry::builder("about")
    .activate(|_, _,_ | {
        println!("About was pressed")
    }).build();

let quit = gio::ActionEntry::builder("quit")
    .activate(|app, _, _| app.quit())
    .build();
app.add_action_entries([about, quit]).unwrap();
```

please re-format the code after updating to the suggested one